src/core.test.ts

```diff
@@ -124,7 +124,7 @@ describe("route generation", () => {
       // getPageRoutes
       existsSyncMock.mockImplementationOnce(() => true);
       findFilesMock.mockReturnValueOnce(["pages/404.ts"]);
-      writeNextJSRoutes({ outDir: "src" });
+      writeNextJSRoutes({ outDir: "src/@types" });
```

Comment:
```suggestion
      writeNextJSRoutes({ outDir: "src" });
```

This should remain as written to test the user override case: https://github.com/tatethurston/nextjs-routes#configuration